### PR TITLE
Fix default path of chain:export command

### LINKED ISSUE #997: Error in chain export
### What happened?

I ran
```
ironfish chain:export
```
It showed me the output
```
Exporting chain from 1 -> 40987 to /usr/src/ironfish-graph-explorer/src/data.json
Exporting blocks: [========================================] 40987/40987 100% | ETA: 0s
    Error: ENOENT: no such file or directory, open '/usr/src/ironfish-graph-explorer/src/data.json'
    Code: ENOENT
```

### Version

ironfish-cli/0.1.21 linux-x64 node-v16.13.0

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by Kupuyc**
## Summary
Fixes #997

Default path of `ironfish chain:export` command is constant and relative - `../ironfish-graph-explorer/src/data.json`. It means command will always fail except when called from `~/ironfish/*` subfolder (in situation when `-p` flag is not set). It possible to fix such behavior using `process.env.PWD`.

## Testing Plan
There is no test for this command - it should be tested manually. Run `ironfish chain:export` w/o `-p` flag from different places.

## Breaking Change
Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by mat-if** [comment]
Checked this out a bit, I'm still seeing the issue. Testing on a mac, `PWD` seems to just return the directory from which the command was executed, not sure if this is different on other operating systems. Compared with `__dirname` just to test:

image

`__dirname` docs for reference https://nodejs.org/docs/latest/api/modules.html#__dirname

**Comment by Kupuyc** [comment]
Yeah... probably it is the best way. Let's ask team members in discord next monday.

File: ironfish-cli/src/commands/chain/export.ts [hunk 1]
```diff
@@ -18,7 +18,11 @@ export default class Export extends IronfishCommand {
       char: 'p',
       parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
       required: false,
-      default: '../ironfish-graph-explorer/src/data.json',
+      default: (): Promise<string> => {
+        Assert.isNotUndefined(process.env.PWD)
+
+        return Promise.resolve(`${process.env.PWD}/../ironfish-graph-explorer/src/data.json`)
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/chain/export.ts:24]: This PR still uses incorrect relative paths. How about we just set the default path to the data directory in a folder called exports? I would recommend maybe just removing this default entirely, then below use the data dir from `this.sdk.config.dataDir` where the data dir is stored if no path is specified.

Comment (Kupuyc) [reply ironfish-cli/src/commands/chain/export.ts]: It's done by second way: default value of flag set to empty string and we will resolve path using `this.sdk.config.dataDir` in that case.

File: ironfish-cli/src/commands/chain/export.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default class Export extends IronfishCommand {
       char: 'p',
       parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
       required: false,
-      default: '../ironfish-graph-explorer/src/data.json',
+      default: '',
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/chain/export.ts:21]:
```suggestion
```

**Review by NullSoldier — APPROVED** [review]

File: ironfish-cli/src/commands/chain/export.ts (hunk 2 above)
Comment (Kupuyc) [reply ironfish-cli/src/commands/chain/export.ts]: Done.
Also, I added path accessibility verification.

**Review by NullSoldier — APPROVED** [review]

**Review by NullSoldier — APPROVED** [review]

**Comment by NullSoldier** [comment]
I just removed the check access, and just had it recursively create the folder for better usability. It's now good to go.

**Pull request merged**